cypress/integration/entitybrowser_spec.js

```diff
@@ -135,12 +135,52 @@ describe('Entity Browser', () => {
 
     it('should load and display essential parts of the list view', () => {
       cy.contains('Daten werden geladen...', {timeout: LONG_TIMEOUT})
-      cy.get('[title="Vorname"]')
+      cy.get('#header-cell-firstname')
```

Comment:
because i have an ID that i use anyway. but you are right. its probably more consistent to only work with data-cy selectors in cypress. will change this